src/Header/Logger.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+
+#ifndef GROUP4_SRC_LOGGER_H_
+#define GROUP4_SRC_LOGGER_H_
+
+#include <string>
+#include <memory>
+#include "OperatorData.h"
+
+// Write operation and result log
+class Logger
+{
+    
+public:
+    
+    // Singleton code convention method
+    static unique_ptr<Logger> GetInstance();
```

Comment:
Please use raw pointer for singleton get method. There can only be one copy of unique_ptr so it won't work for singleton.